WordPress/gutenberg
# Organize gallery controls

## What?
Following up on https://github.com/WordPress/gutenberg/pull/57477 by renaming and ordering the controls to better support visual and logical control hierarchy.  Moves "Resolution" above "Link to" as its a higher priority control. And moves "Open images in new tab" to append to the controls, rather than displace. 

## How?


## Testing Instructions

1. Open a post or page. 
2. Insert a gallery block. 
3. Open block inspector. 
4. See changes.

## Screenshots or screencast 


| Before  | After |
| ------------- | ------------- |
|![CleanShot 2024-01-29 at 14 41 34](https://github.com/WordPress/gutenberg/assets/1813435/3e9ad62e-0660-43fe-ad68-6b3e73431bb3)|![CleanShot 2024-01-29 at 14 44 04](https://github.com/WordPress/gutenberg/assets/1813435/d9103b9b-a95a-4aa5-89e0-7e965c365212)|
